pkg/key/key.go

```diff
@@ -15,6 +15,9 @@ const (
 	DefaultReadAllPermissionsName     = "read-all"
 	DefaultWriteAllPermissionsName    = "write-all"
 	DefaultNamespaceName              = "default"
+	FluxCRDRoleBindingName            = "flux-crd-controller"
+	FluxReconcilerRoleBindingName     = "flux-namespace-reconciler"
+	UpstreamFluxCRDClusterRole        = "crd-controller"
```

Comment:
Both the variable names and the resource names stand out from the existing ones. Is there a good reason for that?

Variable for role binding resource names are:

```nohighlight
WriteOrganizationsPermissionsName
WriteFluxResourcesPermissionsName
WriteClustersPermissionsName
WriteNodePoolsPermissionsName
WriteClientCertsPermissionsName
WriteSilencesPermissionsName
```

Resource names are

```
"write-organizations"
"write-flux-resources"
"write-clusters"
"write-nodepools"
"write-client-certificates"
"write-silences"
```

Reply:
We link to resource names created by upstream Flux. I wouldn't want to rename them, because we want to provide Flux for customer "as is".

Reply:
Wow. I dislike `crd-controller` especially, since it doesn't even have a flux prefix in the name.

What's the purpose of that role? Can we contribute to upstream to change that?

Reply:
That's a good idea. Noted: https://github.com/giantswarm/giantswarm/issues/20732